spec/mongoid/enum_spec.rb

```diff
@@ -55,6 +55,16 @@ class User
       expect(klass::STATUS).to eq values
     end
   end
+  
+  describe "mapping" do
+    it "is should be returns mappings hash" do
+      expect(klass.statuses).to eq({awaiting_approval: 0, approved: 1, banned: 2})
```

Comment:
Line is too long. [82/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.